example/src/main/scala/example/PlainTextBenchmarkServer.scala

```diff
@@ -36,6 +36,7 @@ object Main extends App {
       Server.error(_ => UIO.unit) ++
       Server.disableLeakDetection ++
       Server.consolidateFlush ++
-      Server.disableFlowControl
+      Server.disableFlowControl ++
+      Server.disableObjectAggregator
```

Comment:
```suggestion
      Server.objectAggregator(Int.MaxValue)
```
To enable object aggregation. 
Also add builder methods on Server.